src/runtime_src/xdp/profile/database/static_info/filetypes/aie_control_config_filetype.cpp

```diff
@@ -126,6 +126,32 @@ AIEControlConfigFiletype::getValidKernels() const
     return kernels;
 }
 
+std::vector<std::string>
+AIEControlConfigFiletype::getValidBuffers() const
+{
+    if (getHardwareGeneration() == 1) 
+        return {};
+        
+    std::vector<std::string> buffers;
+
+    // Grab all shared buffers
+    auto sharedBufferTree =
+        aie_meta.get_child_optional("aie_metadata.TileMapping.SharedBufferToTileMapping");
+    if (!sharedBufferTree) {
+        xrt_core::message::send(severity_level::info, "XRT", 
+            getMessage("TileMapping.SharedBufferToTileMapping"));
+        return {};
+    }
+
+    // Now parse all shared buffers
+    for (auto const &shared_buffer : sharedBufferTree.get()) {
+        std::string bufferStr = shared_buffer.second.get<std::string>("bufferName");
+        auto nameStr = bufferStr.substr(bufferStr.find_last_of(".") + 1);
```

Comment:
Do we need this way checking specified buffer name for memory Tiles as well ?  `getMemoryTiles()`